gateway/src/main/kotlin/ru/itmo/lms/gateway/GatewayApplication.kt

```diff
@@ -1,9 +1,14 @@
 package ru.itmo.lms.gateway
 
 import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.autoconfigure.jooq.JooqAutoConfiguration
 import org.springframework.boot.runApplication
 
-@SpringBootApplication
+@SpringBootApplication(
+    exclude = [
+        JooqAutoConfiguration::class,
```

Comment:
Without this it is not working :) As I understand we provide this configuration from `r2dbc` connection poll.